resources/config.rb

```diff
@@ -49,12 +49,12 @@
           default: lazy { node['hostname'] }
 
 property :owner, String,
-          description: 'File/folder group',
-          default: lazy { nginx_user }
+          description: 'File/folder user',
+          default: 'root'
```

Comment:
@hrak haven't we changed it to run *as* root here? Or am I misreading?

Reply:
The latter. The description already gives it away. Property `owner` is used for the ownership of config files and folders. `process_user` and `process_group` is whats used as the user/group the process runs as after dropping privileges.

Reply:
Ohhh ok ok, yes. This is why this looked good to me; it breaks out _config file ownership_ from _what user runs NGINX_ and hardens security of the config files.

Hopefully I didn’t miss something in my review though 😟